src/overcooked_demo/README.md

```diff
@@ -36,13 +36,23 @@ In order to kill the production server, run
 ./down.sh
 ```
 
+You can also start the server via the command line. After installing the `overcooked_ai` via pip, you can start the server by typing
+
+```
+overcooked-demo-up
+```
+
+in the terminal. The same arguments still apply
+
 ## Dependencies
 
-The Overcooked-Demo server relies on the [overcooked-ai](https://github.com/HumanCompatibleAI/overcooked_ai) repo, specifically the submodules overcooked_ai_py and human_aware_rl. Changes made in these modules will be reflected in the overcooked_demo server
+The Overcooked-Demo server relies on the [overcooked-ai](https://github.com/HumanCompatibleAI/overcooked_ai) repo, specifically the submodules overcooked_ai_py and human_aware_rl. Changes made in these modules will be reflected in the overcooked_demo server.
+
+One thing to note is that the main logic depends on submodules in the repo on Github, not the one installed locally. Only changes made in the overcooked_demo locally will be reflected in game
```

Comment:
The docker container [gets the overcooked_ai files from GitHub](https://github.com/HumanCompatibleAI/overcooked_ai/blob/7d561ce0984b9c4889ff7b5c9aee4414b593ed12/src/overcooked_demo/server/Dockerfile#L18) rather than copying local files. I believe it was so the users didn't have to install overcooked_ai to run demo. If users add their own agents to the demo locally they would show up in the web demo, but if the users for some reason want to change the MDP transitions (which is pretty much the only thing demo needs from the overcooked_ai repo) by modifying local files, the changes won't be reflected.

Given that the demo is only supposed to be used for data collection and not for training, I thought there is no need for it to have access to custom env mdp configurations, and this way demo also always work with the newest version of overcooked_ai by default.

Reply:
Yeah that makes sense! 

Might be worth rephrasing this still: "One thing to note is that the main logic depends on submodules in the repo on Github, not the one installed locally. Only changes made in the overcooked_demo locally will be reflected in game"

My understanding from the above is that you meant to say something like: "One thing to note is that local changes to the game logic will not be present by default in the demo (it is pulled directly from the GitHub version). Only changes made in the overcooked_demo locally will be reflected in the demo."